README.md

```diff
@@ -62,6 +62,18 @@ Occasionally, a component will pull a style value from `src/theme/defaultTheme.j
 
 See below sections for high level description of each major component.
 
+### Message Receipts
+
+Render and send read/delivered message receipts with feature enabled in [your connect instance](https://docs.aws.amazon.com/connect/latest/adminguide/message-receipts.html). Follow the changes below to update chat-interface. 
```

Comment:
I think here we are trying to tell users how message receipt works in this repo, but `Follow the changes below to update chat-interface` seems to ask them to make some changes so that they can use message receipt?